### PR TITLE
Feature/v1.13.0-update

### PR DESCRIPTION
**Pull request by carbon-portkey**

### CODE REVIEW

File: packages/mobile-app-did/js/components/ActivityItem/index.tsx
Comment (Portkey-David) [inline packages/mobile-app-did/js/components/ActivityItem/index.tsx:1]: import fonts from 'assets/theme/fonts';
...fonts.mediumFont


bold font use this ,it is adjust android and iOS

File: packages/mobile-app-did/js/components/DepositButton/index.tsx
Comment (Portkey-David) [inline packages/mobile-app-did/js/components/DepositButton/index.tsx:1]: recommond use Constant

File: packages/mobile-app-did/js/pages/DashBoard/Card/index.tsx
Comment (Portkey-David) [inline packages/mobile-app-did/js/pages/DashBoard/Card/index.tsx:1]: the count is variable

File: packages/mobile-app-did/js/pages/Token/TokenDetail/style.ts
Comment (Portkey-David) [inline packages/mobile-app-did/js/pages/Token/TokenDetail/style.ts:1]: use pTd ，when define width

File: packages/mobile-app-did/js/pages/Token/TokenDetail/index.tsx [hunk 1]
```diff
@@ -124,31 +134,25 @@ const TokenDetail: React.FC = () => {
...
-  );
-
-  const isFaucetButtonShow = useMemo(
-    () => !isMainnet && tokenInfo.symbol === defaultToken.symbol && tokenInfo.chainId === 'AELF',
-    [defaultToken.symbol, isMainnet, tokenInfo.chainId, tokenInfo.symbol],
-  );
+  // const isBuyButtonShow = useMemo(
+  //   () => SHOW_RAMP_SYMBOL_LIST.includes(tokenInfo.symbol) && tokenInfo.chainId === 'AELF' && isRampShow,
+  //   [isRampShow, tokenInfo.chainId, tokenInfo.symbol],
+  // );
```
Comment (wade-portkey) [inline packages/mobile-app-did/js/pages/Token/TokenDetail/index.tsx:140]: remove unused code

File: packages/mobile-app-did/js/pages/Token/TokenDetail/index.tsx [hunk 2]
```diff
@@ -124,31 +134,25 @@ const TokenDetail: React.FC = () => {
...
 
-  const isETransToken = useMemo(() => ETransTokenList.includes(tokenInfo.symbol), [tokenInfo.symbol]);
+  // const isFaucetButtonShow = useMemo(
+  //   () => !isMainnet && tokenInfo.symbol === defaultToken.symbol && tokenInfo.chainId === 'AELF',
+  //   [defaultToken.symbol, isMainnet, tokenInfo.chainId, tokenInfo.symbol],
+  // );
```
Comment (wade-portkey) [inline packages/mobile-app-did/js/pages/Token/TokenDetail/index.tsx:145]: remove unused code

**Pull request merged**